src/pbft/pbfttypes.h

```diff
@@ -10,6 +10,12 @@ namespace pbft
   using Term = uint64_t;
   using NodeId = uint64_t;
   using Node2NodeMsg = uint64_t;
+  using CallerId = uint64_t;
+
+  struct Callbacks
```

Comment:
This struct is not being used anymore and it can be removed

Reply:
Ok, will take it out.